gronx.go

```diff
@@ -25,10 +25,13 @@ var expressions = map[string]string{
 	"@10minutes": "*/10 * * * *",
 	"@15minutes": "*/15 * * * *",
 	"@30minutes": "0,30 * * * *",
+
+	"@everysecond": "* * * * * *",
 }
 
 // SpaceRe is regex for whitespace.
 var SpaceRe = regexp.MustCompile(`\s+`)
+var yearRe = regexp.MustCompile(`\d{4}`)
```

Comment:
This regex does not cover all cases. E.g. `0 0 0 ? * * 2023-2025` means at 00:00:00am, every month, between 2023 and 2025

Reply:
it does not have boundary (`^`, `$`) in regex so does cover as it needs to match 4 digits at least and each of `2023` and `2023-2025`, `2023,2024,2025` has 4 consecutive digits somewhere

Reply:
Yes, you're right. Somehow I misinterpreted it.

Reply:
There is one other use case which is `*`